### DISCUSSION TITLE
How send only masked value on submit?

### DISCUSSION THREAD

**Discussion by guuhgalvao** [beholdr/maska#165] [Q&A]
When form is submited i receive a json string with masked|unmasked|completed options, but i want send masked value. 

How i do it?

**Comment by jboada** [comment]
Hi @guuhgalvao ,

if you are able to get the values masked and masked from maska ( [events](https://beholdr.github.io/maska/#/?id=events) ), you are able to set the masked value in a variable in JS and/or hidden input in the html.

From that point you could put the value in a form.

If you had a sandbox, I could help you.

Best Regards,
JB

**Comment by guuhgalvao** [comment]
Hey @jboada, 

Is there no other way?

I'm working with Filament (https://filamentphp.com/), so I need the value sent to be the masked one to avoid unnecessary modifications.

Thanks.
